### PR TITLE
Added description to the adder1 function

### PR DESCRIPTION
**Pull request by t-sasatani**

### CODE REVIEW

**Comment by t-sasatani** [comment]
Sorry for the mess. I just added several comments for preparing for the workshop and noticed that the comment style was wrong, so I added a commit later.
I confirmed that the 9a1ce9c version runs properly on google collaboratory.

File: xls-workshop-openlane.ipynb [hunk 1]
```diff
@@ -300,6 +300,16 @@
       "source": [
         "%%bash -c 'cat > user_module.x; interpreter_main user_module.x'\n",
         "\n",
+        "// The first line declares a function (fn) named \"adder1\" (Functions: https://google.github.io/xls/dslx_reference/#functions)\n",
```
Comment (proppy) [inline xls-workshop-openlane.ipynb:303]: can you add the Japanese version too of those too?
https://github.com/chipsalliance/silicon-notebooks/blob/main/xls-workshop-openlane-ja.ipynb

**Comment by proppy** [comment]
Can you squash your commits? (`git rebase -i` and then `s`)

File: xls-workshop-openlane.ipynb (hunk 1 above)
Comment (t-sasatani) [reply xls-workshop-openlane.ipynb:303]: Yup, done with it.

**Comment by t-sasatani** [comment]
Thanks done squashing @proppy

**Pull request merged**